### PR TITLE
Add "DB Nuke" button for mobile

### PR DESCRIPTION
**Pull request by akalin-keybase**
Add a new Advanced tab in settings and put it there.

### CODE REVIEW

File: shared/settings/advanced/container.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// @flow
+import {TypedConnector} from '../../util/typed-connect'
+import {navigateAppend} from '../../actions/route-tree'
+import DBNuke from './index'
+
+const connector = new TypedConnector()
```
Comment (akalin-keybase) [inline shared/settings/advanced/container.js]: I copied this from `settings/delete`. Do we prefer TypedConnector over the normal connect / mapStateToProps / mapDispatchToProps / mergeProps stuff?

File: shared/settings/advanced/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+function DBNuke(props: Props) {
+  return (
+    <Box
+      style={{
+        ...globalStyles.flexBoxColumn,
+        alignItems: 'center',
+        flex: 1,
+        padding: globalMargins.medium,
+        paddingTop: globalMargins.xlarge,
```
Comment (akalin-keybase) [inline shared/settings/advanced/index.js]: Copied the styling from the desktop "Delete Me" flow. Looks ok to me on mobile, but let me know if I should change it.

File: shared/settings/advanced/container.js (hunk 1 above)
Comment (chrisnojima) [reply shared/settings/advanced/container.js]: nope! this has actually been removed and is very bad for performance

Comment (akalin-keybase) [reply shared/settings/advanced/container.js]: Fixed!

**Pull request merged**